bugbug/bug_features.py

```diff
@@ -287,6 +287,16 @@ def __call__(self, bug, author_ids, **kwargs):
         return bug_reporter()(bug).strip() in author_ids
 
 
+class had_severity_enhancement(object):
```

Comment:
Could you check for how many bugs this is true?

You can do that easily by writing a separate script where you do something like:
```
count = 0
for bug in bugzilla.get_bugs():
    for history in bug['history']:
        ... add 1 to count if had enhancement

print(count)
```

Reply:
The result is 2257.